eng/PatchConfig.props

```diff
@@ -12,4 +12,10 @@
     </PackagesInPatch>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(VersionPrefix)' == '2.1.10' ">
```

Comment:
This will conflict w/ my changes to this file for the 2.1.10 branding but I'll deal with it.